lib/dish.rb

```diff
@@ -0,0 +1,15 @@
+class Dish
+
+  attr_reader :description, :price, :quantity
+
+  def initialize (description, price, quantity)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis.